### PR TITLE
SR paperwork templates

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
- Added 8 paper templates for SR with forms for: employment contract, employee's pay history, fund requisition form, authorization contract, shuttle title change form, derelict ship form, contraband turnover form, contraband permit form (in case no sheriff on duty).
- Mapped SR folder with said templates to SR's office.

## Why / Balance
QoL

## How to test
1. Spawn as SR, look for green folder in the office, inspect papers inside that folder.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl: erhardsteinhauer
- add: Added a folder with paperwork templates to SR's office.

### CODE REVIEW

**Comment by dvir001** [comment]
Based

File: Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_sr.yml [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    - id: PaperWrittenFrontierFormContrabandPermit
+    - id: Paper
+      prob: 0.5
+    - id: Paper
+      prob: 0.5
+
+# Forms
+- type: entity # Based on form made by Willzile1 (Jon-Gah)
+  id: PaperWrittenFrontierFormEmployment
+  name: employment contract
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_sr.yml:36]: Thoughts on capitalizing these names to match the title?  I know it's just a form, but it's still titled.

**Review by whatston3 — CHANGES_REQUESTED** [review]
Removed a bunch of redundancy (do you really need to write out the name of the SR in the form when their signature is at the bottom) and fixed some inconsistent tone.

I've put my rework up on my repo, changes can be seen here: https://github.com/ErhardSteinhauer/frontier-station-14/compare/2024-07-SR-Paperwork-Forms...whatston3:2024-07-24-SR-Paperwork-Forms-suggestions?expand=1

Not entirely happy with the changes, but language should be reasonably consistent between the forms.  Will also need to give space law a quick perusal before signing off on this, but it's looking good.

File: Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_sr.yml [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+        [bold]Time of issuing:[/bold] 
+
+
+        This Permit allows the Holder and his crew,[italic] as written above[/italic], to hold and use C2 Contraband.
+        Non-compliance with Contraband Turnover orders void this Permit.
+        [bold]Class 2 Contraband contains:[/bold] [color=red]all explosive devices not otherwise covered by higher classifications; objects which can be used to non-destructively or non-invasively gain unauthorized access to Nanotrasen secured areas; exclusively restricted and/or authorized use weapons or equipment, uniforms and EMPs; and, security vessels.[/color]
+
+
+
+        [color=#AAAAAA]This Form is void without signs of both Parties and a stamp of NFSD acting Sheriff[/color]
```
[the remark this answers is not in the document]
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_sr.yml:272]: I think SR can issue contraband permits if there is no sheriff around. Might be wrong though.

File: Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_sr.yml (hunk 1 above)
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_sr.yml:36]: None. If you think it'll be better that way, then sure.

**Comment by ErhardSteinhauer** [comment]
> Removed a bunch of redundancy (do you really need to write out the name of the SR in the form when their signature is at the bottom) and fixed some inconsistent tone.
> 
> I've put my rework up on my repo, changes can be seen here: https://github.com/ErhardSteinhauer/frontier-station-14/compare/2024-07-SR-Paperwork-Forms...whatston3:2024-07-24-SR-Paperwork-Forms-suggestions?expand=1
> 
> Not entirely happy with the changes, but language should be reasonably consistent between the forms. Will also need to give space law a quick perusal before signing off on this, but it's looking good.
> 
> Edit: also removed references to "acting" in the valid clause, because forms may still be considered valid by future sheriffs or station reps.

Good stuff, you should merge it. Actually, is there a way for me to merge it?

**Comment by whatston3** [comment]
> Good stuff, you should merge it. Actually, is there a way for me to merge it?

If you have my repo as a remote, you can merge it into your branch.  Assuming you have it checked out and don't have my repo as a remote:
```
git remote add whatstone https://github.com/whatston3/frontier-station-14.git
git merge whatstone/2024-07-24-SR-Paperwork-Forms-suggestions
```

**Comment by ErhardSteinhauer** [comment]
> > Good stuff, you should merge it. Actually, is there a way for me to merge it?
> 
> If you have my repo as a remote, you can merge it into your branch. Assuming you have this branch checked out and don't have my repo as a remote:
> 
> ```
> git remote add whatstone https://github.com/whatston3/frontier-station-14.git
> git merge whatstone/2024-07-24-SR-Paperwork-Forms-suggestions
> ```
> 
> The branch name can be copied nicely from GitHub URLs, which is convenient.

I'm getting `not something we can merge` upon attempt to execute that command

**Review by whatston3 — APPROVED** [review]

**Pull request merged**